distribution/pom.xml

```diff
@@ -213,5 +211,82 @@
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-azure-extensions</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-cassandra-storage</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-cloudfiles-extensions</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-distinctcount</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-rocketmq</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-google-extensions</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-kafka-eight-simple-consumer</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:graphite-emitter</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-orc-extensions</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-parquet-extensions</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-rabbitmq</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:scan-query</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:sqlserver-metadata-storage</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:statsd-emitter</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-thrift-extensions</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-time-min-max</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-virtual-columns</argument>
```

Comment:
@himanshug can you also add ${druid.distribution.pulldeps.opts} as last argument ? 
https://github.com/druid-io/druid/pull/3812